components/script/dom/htmlformelement.rs

```diff
@@ -465,6 +466,22 @@ impl HTMLFormElement {
     /// https://html.spec.whatwg.org/multipage/#statically-validate-the-constraints
     fn static_validation(&self) -> Result<(), Vec<FormSubmittableElement>> {
         let node = self.upcast::<Node>();
+        for child in node.traverse_preorder() {
+            if let Some(el) = child.downcast::<Element>() {
+                if el.disabled_state() {
+                    continue;
+                }
+                let validatable = match el.as_maybe_validatable() {
+                    Some(v) => v,
+                    None => continue,
+                };
+                if !validatable.is_instance_validatable() {
+                    continue;
+                }
+                // pass empty flags initially and set appropriate flags according to the type of element called
+                validatable.validate(ValidationFlags::empty());
+            }
+        }
```

Comment:
hello

Pushing as a new commit with unsuccessful compilation, that will create issues. If you are ok, I can share the portion of code file here by attaching a document and also mention what kind of error I am getting. Actually error is related to rust syntax and I believe its nothing major.

Also, if you are ok sharing your email ID, I can email you the details.

Regards
Bhavya